src/cpp/server/ServerMain.cpp

```diff
@@ -622,6 +624,11 @@ int main(int argc, char * const argv[])
       if (error)
          return core::system::exitFailure(error, ERROR_LOCATION);
 
+      // initialize database connectivity
+      error = server_core::database::initialize(true);
```

Comment:
Do we ever initialize the server database without attempting to update the schema? If not, is the parameter there because we plan to in the future?

Reply:
We don't yet, but we will. I want derivative processes of rserver to not run migrations, even though they may need to connect to the database (like `rserver-monitor`).